util/log/clog.go

```diff
@@ -510,7 +510,7 @@ func formatHeader(s Severity, now time.Time, threadID int32, file string, line i
 	n += buf.nDigits(6, n, now.Nanosecond()/1000, '0')
 	tmp[n] = ' '
 	n++
-	n += buf.nDigits(7, n, int(threadID), ' ')
+	n += buf.someDigits(n, int(threadID))
```

Comment:
Does it even make sense to keep logging this field? It feels silly to log the same value on every line, although I do find it occasionally useful. Will we ever resort to the hacks necessary to get a "thread ID" in Go?